team-config.yml

```diff
@@ -168,3 +168,21 @@ opal:
     build_notices_channel: "#opal-builds"
   tags:
     application: opal
+pdda:
+  team: "PDDA"
+  namespace: "pdda"
+  azure_ad_group: DTS SDS PDDA
+  slack:
+    contact_channel: "#pdda"
+    build_notices_channel: "#pdda-builds"
+  tags:
+    application: public-display-data-aggregator
+pdm:
+  team: "PDM"
+  namespace: "pdm"
+  azure_ad_group: DTS SDS PDM
+  slack:
+    contact_channel: "#pdm"
+    build_notices_channel: "#pdm-builds"
```

Comment:
these slack channels don't seem to exist or are private and should not be

Reply:
Apologies.
Those 2 slack channels have now been created and are public.